transports/janus_mqtt.c

```diff
@@ -720,48 +780,13 @@ int janus_mqtt_client_connect(janus_mqtt_context *ctx) {
 }
 
 void janus_mqtt_client_connect_success(void *context, MQTTAsync_successData *response) {
-	janus_mqtt_client_connect_success_impl(context);
```

Comment:
Please don't keep methods empty. At the very least, there should be a comment explaining why we don't care about this callback.